exporter/datadogexporter/factory.go

```diff
@@ -34,11 +35,18 @@ import (
 	"google.golang.org/protobuf/proto"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/hostmetadata"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/logsagent"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter/internal/metadata"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/datadog"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry"
 )
 
+var logsAgentExporterFeatureGate = featuregate.GlobalRegistry().MustRegister(
+	"exporter.datadogexporter.logsagentexporter",
+	featuregate.StageAlpha,
+	featuregate.WithRegisterDescription("When enabled, logs export in datadogexporter uses the new logs agent exporter."),
```

Comment:
Reword `When enabled, logs export in datadogexporter uses the new logs agent exporter.` to 
`When enabled, datadogexporter uses the Datadog agent logs pipeline for exporting logs`.